### PR TITLE
Release v0.10.4

### PR DESCRIPTION
**Pull request by bassosimone**
This PR contains some commits that are meant to be included in v0.10.4.

### CODE REVIEW

**Comment by bassosimone** [comment]
I've created a tag off this branch: https://github.com/measurement-kit/measurement-kit/releases/tag/v0.10.4.

Will soon merge and build all the binaries.

File: include/measurement_kit/README.md [hunk 1]
```diff
@@ -919,7 +946,7 @@ by setting the appropriate settings.
   let probe_cc = "ZZ"
   if (settings.options.probe_cc != "") {
     probe_cc = settings.options.probe_cc
-  } else if (settings.options.geoip_country_path != "") {
+  } else if (!settings.options.no_geoip) {
```
Comment (lorenzoPrimi) [inline include/measurement_kit/README.md]: I like it!

File: include/measurement_kit/internal/vendor/mkcollector.hpp [hunk 2]
```diff
@@ -21,8 +21,9 @@ class Settings {
   /// ca_bundle_path is the path to the CA bundle (required on mobile)
   std::string ca_bundle_path;
 
-  /// timeout is the whole operation timeout (in seconds)
-  int64_t timeout = 30;
+  /// timeout is the whole operation timeout (in seconds). Zero indicatest
+  /// that there actually is no timeout.
+  int64_t timeout = 0;
```
Comment (lorenzoPrimi) [inline include/measurement_kit/internal/vendor/mkcollector.hpp]: so we leave the timeout managment entirely to the app?

Comment (bassosimone) [reply include/measurement_kit/internal/vendor/mkcollector.hpp]: Yes. In all the cases where a default timeout was a bad idea, I zeroed the timeout. Which means that, if the app sets no timeout, there's no timeout!

Comment (lorenzoPrimi) [reply include/measurement_kit/internal/vendor/mkcollector.hpp]: What is the implication of a no timeout? the operation can run forever?

Comment (bassosimone) [reply include/measurement_kit/internal/vendor/mkcollector.hpp]: Yes, it's up to the app to set a timeout. I think, if the app does not set one, the user would eventually be bored and kill the app. I'm designing a better scheme in Go where the user would be able to interrupt a pending operation. Theoretically, I could also implement it in C++, but I think it's probably best to focus on Go at this point, as it's definitely easier.

Comment (lorenzoPrimi) [reply include/measurement_kit/internal/vendor/mkcollector.hpp]: Ok I'm gonna check that actually there is a timeout everywhere in the app and approve this PR

**Pull request merged**